src/Commands/InstallCommand.php

```diff
@@ -77,13 +77,15 @@ public function handle(Filesystem $filesystem)
         $this->call('migrate', ['--force' => $this->option('force')]);
 
         $this->info('Attempting to set Voyager User model as parent to App\User');
-        if (file_exists(app_path('User.php'))) {
-            $str = file_get_contents(app_path('User.php'));
+        if (file_exists(app_path('User.php')) || file_exists(app_path('Models/User.php'))) {
+            $userPath = file_exists(app_path('User.php')) ? app_path('User.php') : app_path('Models/User.php');
+
+            $str = file_get_contents($userPath);
 
             if ($str !== false) {
                 $str = str_replace('extends Authenticatable', "extends \TCG\Voyager\Models\User", $str);
 
-                file_put_contents(app_path('User.php'), $str);
+                file_put_contents($userPath, $str);
             }
         } else {
             $this->warn('Unable to locate "app/User.php".  Did you move this file?');
```

Comment:
Let's as a `or app/Models/User` here.

Reply:
What do you mean? `$userPath` already contains the correct path

Reply:
I mean the warning shown if no model is found

Reply:
oh, got it! I'll fix it later